psi4/src/psi4/libfock/DirectJK.cc

```diff
@@ -385,7 +385,7 @@ void DirectJK::compute_JK() {
         timer_on("DirectJK: INCFOCK Preprocessing");
         incfock_setup();
         int reset = options_.get_int("INCFOCK_FULL_FOCK_EVERY");
-        double dconv = options_.get_double("D_CONVERGENCE");
+        double dconv = options_.get_double("INCFOCK_CONVERGENCE");
         double Dnorm = Process::environment.globals["SCF D NORM"];
         // Do IFB on this iteration?
         do_incfock_iter_ = (Dnorm >= dconv) && !initial_iteration_ && (incfock_count_ % reset != reset - 1);
```

Comment:
Actually people reading this code might misread "dconv" to mean the user-desired convergence on the density, which isn't what it means.  So, similarly to changing D_CONVERGENCE to INCFOCK_CONVERGENCE, how about renaming dconv to incfock_conv or something?

Reply:
Fixed it